docs/e2e-testing.md

```diff
@@ -35,9 +35,7 @@ func genericTestSuite(t *testing.T) {
 
     // Create the operatorsource.
     err = helpers.CreateRuntimeObject(f, ctx, helpers.CreateOperatorSource(namespace))
-    if err != nil {
-        t.Errorf("Could not create operator source: %v", err)
-    }
+    require.NoError(t, err, fmt.Sprintf("Could not create OperatorSource: %v", err))
```

Comment:
This is an example of: https://github.com/operator-framework/operator-marketplace/pull/177#discussion_r285264592